packages/core/src/textures/TextureSystem.ts

```diff
@@ -69,6 +70,11 @@ export class TextureSystem extends System
          * @readonly
          */
         this.unknownTexture = new BaseTexture();
+
+        /**
+         * Whether glTexture with int/uint sampler type was uploaded
+         */
```

Comment:
Same thing, can you add this to the top instead of in the constructor? No reason to maintain this legacy format for new members.